tasks/install.yml

```diff
@@ -1,12 +1,16 @@
 ---
 
-- name: "Install 2.x Family"
+- name: Install apt-transport-https
```

Comment:
Is this step needed? If yes then it probably needs become: yes.

Reply:
It should be installed by default, but wasn't on the docker images.  Required for `https://` repos.  Added the `become: yes`.